src/components/accordion/_accordion.scss

```diff
@@ -1,3 +1,5 @@
+@use "../../mixin" as *;
```

Comment:
There is no need to import this in each css file.You can import this in theme.css.

Reply:
Class name can be defined as :
https://github.com/adobe/aem-core-forms-components/blob/master/examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/clientlibs/forms-clientlib-site/styles/components/adaptive-form/wizard.less